### DISCUSSION TITLE
Turkey-Turkcell Works Fine Pixel 7 Pro

### DISCUSSION THREAD

**Discussion by techgenius1** [kyujin-cho/pixel-volte-patch#107] [Carrier-specific Experiences]
Hello, im using Turkcell Operator and VoLTE/VoWiFi/VT all works fine on my Pixel 7 Pro.

But i want to ask something @kyujin-cho ,
I cant use LTE_CA on my phone probably because Pixel's doesnt sold here in Turkey and thats why google doesnt makes a proper carrier config for Turkish operators. Is there anything can you do about enabling LTE_CA support on carrier config side ?
I can only able to use 2xCA and getting around 100 Mbps internet speed because of that, but my old OnePlus 8T can get 4xCA and it gets around 400 Mbps speed

Also i can see that LTE_CA is overridden on `*#*#4636#*#*` menu
![5990127730693357290_120](https://user-images.githubusercontent.com/10501401/230501285-69af8d34-5f7c-458b-a3c1-8bd3b103f0a7.jpg)

**Comment by kyujin-cho** [comment]
Hi! Perhaps LTE+ toggles can do the job. If it does not work... Then out of an idea.

**Reply by techgenius1** [reply]
LTE+ switch is on and was always working but it only gives me 2xCA

**Reply by techgenius1** [reply]
Does that has something to do with user agent ?
